File: features/lender/src/main/java/com/nivasafinance/features/lender/lender/dto/LenderResponseData.java
package com.nivasafinance.features.lender.lender.dto;

import com.nivasafinance.features.lender.lender.enums.LenderStatus;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class LenderResponseData {
    private UUID id;
    private String name;
    private String key;
    private LenderStatus status;
}

File: features/lender/src/main/java/com/nivasafinance/features/lender/lender/dto/LenderResponseData.kt
package com.nivasafinance.features.lender.lender.dto

import com.nivasafinance.features.lender.lender.enum.LenderStatus
import java.util.UUID

data class LenderResponseData(
    val id: UUID,
    val name: String,
    val key: String,
    val status: LenderStatus
)
